temporal/api/cloud/account/v1/message.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package temporal.api.cloud.account.v1;
+
+option go_package = "go.temporal.io/api/cloud/account/v1;account";
+
+message MetricsSpec {
+    // Enables the endpoint from which clients can scrape all their namespace metrics.
+    bool enabled = 1;
+    // The base64 encoded ca cert(s) in PEM format that clients connecting to the metrics endpoint can use for authentication.
+    // This must only be one value, but the CA can have a chain.
+    string accepted_client_ca = 2;
```

Comment:
```suggestion
    bytes accepted_client_ca = 2;
```

Why not just accept the PEM instead of base64 encoded (which is what proto JSON will be anyways FWIW)

Reply:
This was my fault for not checking that well enough. PEM is already mostly base64, it's a bit weird to double encode. But I guess it's up to y'all.

Reply:
If we keep the double-encoding, we should add to the comment about this. Regardless of bytes or strings, I would have expected that `base64 encoded ca cert(s)` means PEM format.

Reply:
if we don't want to change the namespace spec (since I guess it's public preview), the other alternative is to just add another field which takes bytes to both account/namespace and then eventually deprecate the string field?

Reply:
Okay, changed the type to []byte. Which expects the raw pem certs.